biome.jsonc

```diff
@@ -35,13 +35,9 @@
     "rules": {
       "recommended": true,
       "suspicious": {
-        "noAsyncPromiseExecutor": "off",
         "noExplicitAny": "off",
```

Comment:
It would be great to disable this one as well 😄 , but we will have a lot of places to fix it.